desktop/core/src/desktop/urls.py

```diff
@@ -250,7 +250,7 @@
     else:
       namespace = {}
     if namespace or app in appmanager.DESKTOP_APPS:
-      app_urls_patterns.append(url('^' + re.escape(app.name) + '/', include(app.urls, **namespace)))
+      app_urls_patterns.append(url('^' + re.escape(app.name) + '/', include(app.urls)))
```

Comment:
Aren't we missing the notion of namespace name there? (and it will break pages)

Reply:
as given (If the app_name is set in this new way, the namespace argument is no longer required. It will default to the value of app_name) in this https://docs.djangoproject.com/en/2.0/releases/1.9/#passing-a-3-tuple-or-an-app-name-to-include we can remove the namespace.

Reply:
Better to keep it explicit to be 100% sure, it still has `namespace=..` AFAIK?

Reply:
Don't we lose the equivalent of `app_name = 'polls'  # added`?

Did you test the changes?